SEBrowser/Scripts/TSX/Components/TrendData/TrendDataNavbar.tsx

```diff
@@ -148,6 +157,44 @@ const TrendSearchNavbar = React.memo((props: IProps) => {
         });
     }, [channelGroupStatus, phaseStatus]);
 
+    React.useEffect(() => {
+        let range = "";
+        const startMoment = moment(timeFilter.start); // These default to the date+time format
```

Comment:
Please don't use default moment formats, it can lead to unexpected bugs. Be explicit